cypress/e2e/tests/migration/applicationinventory/assessment/miscellaneous.test.ts

```diff
@@ -272,6 +274,31 @@ describe(["@tier3"], "Tests related to application assessment and review", () =>
         applications[0].closeApplicationDetails();
     });
 
+    it("Validates applications that inherit assessment gets displayed on Reports tab", function () {
```

Comment:
@Neilhamza , thanks for the PR. Could you move the test to the 'migration/reports-tab' directory ?

Reply:
done